Remove open php tag

crftwrk opened the pull request:
Open the `<?php` leads into an error, because it's opened already at the top of the `functions.php` and not closed.

crftwrk committed:
Remove open php tag

crftwrk commented:
Hi Anisur,

thank you for merging this PR. We want to include this in our theme https://github.com/bootscore/bootscore. Because you know at best how your scripts works, I would kindly ask if it's possible to hire you to implement it. If so, how much you will charge?

Thanks in advance

Basti

anisur2805 commented:
Hi Basti,

Thank you for reaching out. I'm glad to know that you wanna hire me for
this job. So more clarification can you please describe to me more details
about the actual implementation? Like what's your requirement? And after
that, I'll be able to inform you of the charge amount.

Thank you
Anisur

On Sat, Feb 17, 2024 at 5:14 PM Basti ***@***.***> wrote:

> Hi Anisur,
>
> thank you for merging this PR. We want to include this in our theme
> https://github.com/bootscore/bootscore. Because you know at best how your
> scripts works, I would kindly ask if it's possible to hire you to implement
> it. If so, how much you will charge?
>
> Thanks in advance
>
> Basti
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/anisur2805/woocommerce-mini-cart-ajax-plus-minus/pull/1#issuecomment-1949939840>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACUCF2VPZO6IOYIKCUQRETLYUCGI7AVCNFSM6AAAAABDMI47E6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTSNBZHEZTSOBUGA>
> .
> You are receiving this because you modified the open/close state.Message
> ID: <anisur2805/woocommerce-mini-cart-ajax-plus-minus/pull/1/c1949939840@
> github.com>
>

crftwrk commented:
Hi Ansur,

thank you for your quick response. We have a free Bootstrap theme https://github.com/bootscore/bootscore with an included an Ajax cart.

A user asked for quantity input https://github.com/bootscore/bootscore/issues/631 in mini-cart and I think this is a good idea for the next major release.

Roughly added a sketch for qty input in offcanvas mini-cart here https://dev.bootscore.me/shop/, it shows how qty should work. 

The task is:

- Adding Ajax quantity input to the existing mini-cart.php using the existing -/+ buttons (at best as a hook)
- Quantity badge in navbar cart button should updated as well when changing the quantity
- Updated price for single item if quantity is changed like your example
- Updated Subtotal if qty changed
- All scripts should be done in a single file https://github.com/bootscore/bootscore/blob/main/woocommerce/inc/ajax-add-to-cart.php to allow users quickly to disable Ajax functionality.

Let me know if you need more information.


Thanks


Basti
